<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<HTML>
 <HEAD>
   <TITLE> [mod_python] importing home made modules
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20importing%20home%20made%20modules&In-Reply-To=">
   <META NAME="robots" CONTENT="index,nofollow">
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="015942.html">
   <LINK REL="Next"  HREF="015947.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[mod_python] importing home made modules</H1>
<table border=0 width="100%"><tr><td valign="top">
    <B>Bo Lorentsen</B> 
    <A HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20importing%20home%20made%20modules&In-Reply-To="
       TITLE="[mod_python] importing home made modules">bl at netgroup.dk
       </A><BR>
    <I>Mon Jul 19 19:54:26 EDT 2004</I>
    <P><UL>
        <LI>Previous message: <A HREF="015942.html">[mod_python] Multiple interpreter instances?
</A></li>
        <LI>Next message: <A HREF="015947.html">[mod_python] importing home made modules
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#15945">[ date ]</a>
              <a href="thread.html#15945">[ thread ]</a>
              <a href="subject.html#15945">[ subject ]</a>
              <a href="author.html#15945">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Hi ...

I have a strage problem, while using apache2 2.0.49 and mod_python 3.1.3
(Python 2.3.4) on a debian testing/sarge platform.

I have made a python module written i C/C++ for python that just works
nicly on the command line python, it both imports and works as expected.
but when I include it in a python module via apache2 I get an
segmentation error. Ofcause plain mod_pyton modules without this module
just works :-)

So my module works just fine, but in apache2 :-(

The funny part is, that this workeed for just a few weeks ago, in the
same setup, but due to a hacker attack I was forced to reinstall the
mashine. The only thing I can think of that may have changed is the
python version (2.3.3-&gt;2.3.4).

Are there anyone that have had any problems like this, or will it be
wise to use python 2.2 instead ?

/BL

</PRE>
<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="015942.html">[mod_python] Multiple interpreter instances?
</A></li>
	<LI>Next message: <A HREF="015947.html">[mod_python] importing home made modules
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#15945">[ date ]</a>
              <a href="thread.html#15945">[ thread ]</a>
              <a href="subject.html#15945">[ subject ]</a>
              <a href="author.html#15945">[ author ]</a>
         </LI>
       </UL>

</td>
<td align="right" valign="top">
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-42971867-2', 'modpython.org');
  ga('send', 'pageview');
</script>

</td>
</table>

<hr>
<a href="http://mailman.modpython.org/mailman/listinfo/mod_python">More information about the Mod_python
mailing list</a><br>
</body></html>
